doc/manuals/user/context_providers.md

```diff
@@ -62,7 +62,9 @@ EOF
 ```
       
       
-* Next, consider that a client queries the Street4 temperature
```

Comment:
I understand that this line should not be modified. Or maybe I'm missing something?

Reply:
Yes, I may have made a mistake